index-plants.md

```diff
@@ -26,17 +26,13 @@ The server is part if the European Galaxy server and is maintained by the [RNA B
 # Get started
 
 Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take [a guided tour]({{ page.website }}/tours/core.galaxy_ui){:target="_blank"} through Galaxy's user interface.
+Some relevant materials prior to the analysis of plant-related data is the analysis of [Chloroplast genome assembly](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/chloroplast-assembly/tutorial.html).
+
 
 # Training and Workshops
 
 We are passionate about training. So we are working in close collaboration with the [Galaxy Training Network (GTN)](https://galaxyproject.org/teach/gtn/){:target="_blank"} to develop training materials of data analyses based on Galaxy {% cite batut2017community %}. These materials hosted on the GTN GitHub repository are available online at [https://training.galaxyproject.org](https://training.galaxyproject.org){:target="_blank"}.
```

Comment:
what about adding this link? https://training.galaxyproject.org/training-material/search?query=plant

Reply:
@mtekman can you unify the labels plant and plants? should [this other tutorial](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html) be labelled as plants too?

Reply:
Added it, I moved this down to the main material section

![Screenshot_2021-04-13 Galaxy Europe](https://user-images.githubusercontent.com/20641402/114552626-16052d00-9c65-11eb-91c5-3fdadaaaa812.png)